projects/configs/tag-operations/test-extract-uclh-omop-cdm.yaml

```diff
@@ -607,3 +607,47 @@
   group: 0x000d
   element: 0x1001
   op: "keep"
+- name: "Position Reference Indicator"
```

Comment:
same here for keeping everything in order of group and then element

Reply:
YAML file has been re-ordered here [5d47397](https://github.com/SAFEHR-data/PIXL/pull/493/commits/5d4739721b89c67d5682962e1b29ddcc241834b2)